app.py

```diff
@@ -448,51 +402,51 @@ def check_all_enemies_defeated():
         else:
             next_stage()
 
-def generate_enemies(stage):
-    global enemies
-    enemies.clear()
+# def generate_enemies(stage):
```

Comment:
The "generate_enemies" function was actually in this file two times. I removed the first, but kept this one and commented it all out just in case you needed it for some reason!